src/lamp_py/performance_manager/l1_rt_trips.py

```diff
@@ -374,7 +510,7 @@ def update_trip_stop_counts(db_manager: DatabaseManager) -> None:
 
     process_logger = ProcessLogger("l1_trips.update_trip_stop_counts")
     process_logger.log_start()
-    db_manager.execute(update_query)
+    db_manager.execute(update_query, disable_trip_tigger=True)
```

Comment:
disable trigger that was applying `branch_route_id` and `trunk_route_id` previously, these vestiges of the database functions should be totally removed in a future PR

Reply:
do we turn the trip trigger on anywhere?

Reply:
not anymore